### PR TITLE
versions: Update cloud-hypervisor to release v0.14.1

### PR DESCRIPTION
**Pull request by likebreath**
Highlights for cloud-hypervisor version 0.14.0 include: 
1. Structured event monitoring; 
2. MSHV improvements; 
3. Improved aarch64 platform; 
4. Updated hotplug documentation;
5. PTY control for serial and virtio-console; 
6. Block device rate limiting; 
7. Plan to deprecate the support of "LinuxBoot" protocol and support PVH protocol only.

Highlights for cloud-hypervisor version 0.13.0 include: 
1) Wider VFIO device support; 
2) Improve huge page support; 
3) MACvTAP support; 
4) VHDdisk image support; 
5) Improved Virtio device threading; 
6) Clean shutdown support via synthetic power button.

Details can be found: https://github.com/cloud-hypervisor/cloud-hypervisor/releases

Note: The client code of cloud-hypervisor's OpenAPI is automatically
generated by `openapi-generator` [1-2]. As the API changes do not
impact usages in Kata, no additional changes in kata's runtime are
needed to work with the latest version of cloud-hypervisor.

[1] https://github.com/OpenAPITools/openapi-generator
[2] https://github.com/kata-containers/kata-containers/blob/main/src/runtime/virtcontainers/pkg/cloud-hypervisor/README.md

Fixes: #1591

Signed-off-by: Bo Chen <chen.bo@intel.com>

### CODE REVIEW

**Review by egernst — COMMENTED** [review]
LGTM - thanks for this!

For my own knowledge, can you update commit message or at least add comment in PR on how you made the updates for the package?

**Review by sameo — APPROVED** [review]

**Review by fidencio — APPROVED** [review]
lgtm, thanks @likebreath!

**Comment by fidencio** [comment]
I'm adding the do-not-merge label till @egernst's comment is addressed.

**Review by GabyCT — APPROVED** [review]

File: versions.yaml [hunk 1]
```diff
@@ -75,7 +75,7 @@ assets:
       url: "https://github.com/cloud-hypervisor/cloud-hypervisor"
       uscan-url: >-
         https://github.com/cloud-hypervisor/cloud-hypervisor/tags.*/v?(\d\S+)\.tar\.gz
-      version: "v0.12.0"
+      version: "v0.14.0"
```
Comment (rbradford) [inline versions.yaml:78]: Please use v0.14.1 now.

**Comment by likebreath** [comment]
> LGTM - thanks for this!
> 
> For my own knowledge, can you update commit message or at least add comment in PR on how you made the updates for the package?

@egernst Good point, and comment is addressed. Any other advises/questions, please let me know.

> better to rely on https://github.com/cloud-hypervisor/cloud-hypervisor/tree/stable/v0.14.x as it includes some important stability fixes :)

 @sboeuf @rbradford PR updated. Thanks for the timely inputs!

/test

**Review by jcvenegas — APPROVED** [review]

**Review by egernst — APPROVED** [review]

**Pull request merged**